tests/rados/test_rados_preempt_scrub.py

```diff
@@ -18,210 +19,236 @@
-        while time.time() < time_end:
-            scrub_status = check_scrub_status(rados_object)
-            if scrub_status is False:
-                scrub_status_flag = True
-                log.info("Scrubbing is not in progress.Continuing the test execution")
-                break
-            log.info("Waiting for the scrub to complete")
-            time.sleep(30)
-
-        if scrub_status_flag is False:
-            log.error(
-                "Scrubbing is in progress cannot continue the further execution of test"
-            )
+        # enable the file logging
+        if not rados_object.enable_file_logging():
+            log.error("Error while setting config to enable logging into file")
             return 1
 
-        for entry in pool_target_configs.values():
-            log.debug(
-                f"Creating {entry['pool_type']} pool on the cluster with name {entry['pool_name']}"
-            )
-            method_should_succeed(
-                rados_object.create_pool,
-                **entry,
-            )
-            log.info(f"Created the pool {entry['pool_name']}")
-            rados_object.bench_write(
-                pool_name=entry["pool_name"], rados_write_duration=90
-            )
-            flag = wait_for_clean_pg_sets(rados_object)
-            if not flag:
-                log.error(
-                    "The cluster did not reach active + Clean state after add capacity"
-                )
-                return 1
-            mon_obj.set_config(
-                section="osd", name="osd_shallow_scrub_chunk_max", value="250"
-            )
-            mon_obj.set_config(section="osd", name="osd_scrub_chunk_max", value="250")
-            mon_obj.set_config(section="osd", name="debug_osd", value="10/10")
-
-            rados_object.bench_write(
-                pool_name=entry["pool_name"], rados_write_duration=300, background=True
-            )
-
-            log_lines = [
-                "head preempted",
-                "WaitReplicas::react(const GotReplicas&) PREEMPTED",
-            ]
-
-            init_time, _ = installer.exec_command(cmd="sudo date '+%Y-%m-%d %H:%M:%S'")
-
-            rados_object.run_scrub(pool=entry["pool_name"])
-            scrub_status_flag = False
+        init_time, _ = installer.exec_command(
+            cmd="sudo date '+%Y-%m-%dT%H:%M:%S.%3N+0000'"
+        )
+        init_time = init_time.strip()
+        rados_object.configure_pg_autoscaler(**{"default_mode": "off"})
+        method_should_succeed(rados_object.create_pool, **config)
+
+        if not rados_object.bench_write(
+            pool_name=pool_name, rados_write_duration=20, max_objs=15, num_threads=1
+        ):
+            log.error(f"Failed to write objects into {pool_name}")
+            return 1
+        acting_set = rados_object.get_pg_acting_set(pool_name=pool_name)
+        log.info(f"The PG acting set is -{acting_set}")
+        set_parameter_value(mon_obj, rados_object, pool_name)
```

Comment:
This method is very specific in setting the params for generating "preempt" messages on OSD logs. Let's rename the method to `set_preempt_parameter_value`

Reply:
Modified the method name.